<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioWizard</class>
 <widget class="QWizard" name="AudioWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Tuning Wizard</string>
  </property>
  <widget class="QWizardPage" name="qwpIntro">
   <property name="title">
    <string>Introduction</string>
   </property>
   <property name="subTitle">
    <string>Welcome to the RetroShare Audio Wizard</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>This is the audio tuning wizard for RetroShare. This will help you correctly set the input levels of your sound card, and also set the correct parameters for sound processing in Retroshare. </string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>291</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="qwpVolume">
   <property name="title">
    <string>Volume tuning</string>
   </property>
   <property name="subTitle">
    <string>Tuning microphone hardware volume to optimal settings.</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="qliVolumeTuningText">
      <property name="text">
       <string>&lt;p&gt;Open your sound control panel and go to the recording settings. Make sure the microphone is selected as active input with maximum recording volume. If there's an option to enable a &amp;quot;Microphone boost&amp;quot; make sure it's checked. &lt;/p&gt;
&lt;p&gt;Speak loudly, as when you are annoyed or excited. Decrease the volume in the sound control panel until the bar below stays as high as possible in the green and orange but not the red zone while you speak. &lt;/p&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="qliAmpTuningText">
      <property name="text">
       <string>Talk normally, and adjust the slider below so that the bar moves into green when you talk, and doesn't go into the orange zone.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="qsMaxAmp">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>32767</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>552</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="qcbStopEcho">
        <property name="text">
         <string>Stop looping echo for this wizard</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="qcbHighContrast">
        <property name="toolTip">
         <string>Apply some high contrast optimizations for visually impaired users</string>
        </property>
        <property name="text">
         <string>Use high contrast graphics</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="qwpTrigger">
   <property name="title">
    <string>Voice Activity Detection</string>
   </property>
   <property name="subTitle">
    <string>Letting RetroShare figure out when you're talking and when you're silent.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="qliVADText">
      <property name="text">
       <string>This will help Retroshare figure out when you are talking. The first step is selecting which data value to use.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="qrPTT">
        <property name="text">
         <string>Push To Talk:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string notr="true">todo shortcut</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" rowspan="3">
     <widget class="QLabel" name="qlTalkIcon">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QRadioButton" name="qrVAD">
      <property name="text">
       <string>Voice Detection</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QWidget" name="qwVAD" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="qliVadTuningText">
         <property name="text">
          <string>Next you need to adjust the following slider. The first few utterances you say should end up in the green area (definitive speech). While talking, you should stay inside the yellow (might be speech) and when you're not talking, everything should be in the red (definitively not speech).</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="qsTransmitMin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32767</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="qsTransmitMax">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32767</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="qrContinuous">
      <property name="text">
       <string>Continuous transmission</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="qwpDone">
   <property name="title">
    <string>Finished</string>
   </property>
   <property name="subTitle">
    <string>Enjoy using RetroShare</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="qlDone">
      <property name="text">
       <string>Congratulations. You should now be ready to enjoy a richer sound experience with Retroshare.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>267</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
